src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -599,6 +599,7 @@
   "com.minecolonies.coremod.gui.debugscreen.noclosecolony": "No close colony!",
   "com.minecolonies.coremod.gui.debugscreen.nextcolony": "Next colony is %d blocks away (too close to found a new one)!",
   "com.minecolonies.coremod.gui.debugscreen.blocksfromcenter": "%d blocks from the center.",
+  "com.minecolonies.coremod.debug.citizenloadfailure": "Worker %s has been unassigned from his job, a problem was found during load.",
```

Comment:
Small nitpick: I think that "their job" instead of "his job" is more correct. Unless it has been proven that this can only happen for male citizens, which sounds very unlikely